### DISCUSSION TITLE
Custom Ability Abbreviations

### DISCUSSION THREAD

**Discussion by Yevn** [misthero/dnd5e-custom-skills#14] [Q&A]
After trying to get this module to work and only realizing shortly after that it was because of an incorrect folder name, I've got the module running and it's going fine. There are some lag spikes when trying to add the custom abilities and skills, or sometimes outright freezing (Most likely an issue with my hardware though). 

My question isn't about that though, I'm asking about the abbreviations for the new ability scores. I'm using the Skill customization module, but even with the new skills I create, the abbreviation for the new ability scores only comes up as "DND". I would like to fix this so my players know what scores are being applied. If someone has also experienced this, I would love the help.

**Comment by misthero** [comment]
I never tested the Skill Customization module interactions with this one. I have to check. Are you using default character sheet?

About the lag, do you have many actors? How many? (rougly)

**Reply by Yevn** [reply]
I was using the default character sheet originally, but I had switched over to the tidy sheet. The issue persists with both of them.

I didn't consider the number of actors. I have the whole srd monster compendium set up, so there are a lot of them. That would definitely explain that. I suppose I'll remove them until I'm finished setting up and then re-add them as actors.

**Reply by Yevn** [reply]
I don't know if I'm missing how to do it, but I should also mention that I had to go through a roundabout way to configure one of my new ability scores to Initiative and AC in place of dex by using the DAE module. Idk if you have plans for that, but I wanted to make you aware of that. I guess they technically don't count as skills or abilities, but it'd be awesome to set those up with new scores potentially.

**Reply by misthero** [reply]
I'm back on this, Do you still have isssues? 
I think you are talking about Foundry v9 maybe?
Since I tested and Skill Customization is not working on v10.

Sadly new features and bug fixes will be only on v10.

also I think that skill customization is not needed anymore since you can change bonuses directly in core

![immagine](https://user-images.githubusercontent.com/3662610/189831846-758d4a71-fdbb-4250-909d-b271b22bd2a3.png)

am I wrong?

**Reply by Yevn** [reply]
Sorry it took me a while to respond. No, the issues I'm experiencing are on v10. I didn't realize that you could change skills for checks in v10 without the Skill Customizer. What I'm having an issue with is still present with or without Skill Customization.

![Skill bug](https://user-images.githubusercontent.com/96644234/191912716-672a98eb-a760-4ffd-a24e-4348cd59e6c5.JPG)

This is what I'm talking about specifically. If I try to change the default score used for the skill, the abbreviations for my new ability scores only show up as DND. I'm trying to figure out if that's a bug or where I can change that. 

I need to check again also, but the image you sent is for each specific roll, whereas I thought skill customization would let me change the default score. I had even tried just using your module to make copies of each skill with my scores, but the abbreviations still remain as DND.
